### DISCUSSION TITLE
What is the rule parameter to convert a transaction to a transfer?

### DISCUSSION THREAD

**Discussion by Oliv4945** [firefly-iii/firefly-iii#6856] [Firefly III] [answered]
Hello,

I am trying to convert a withdrawal to a transfer thanks to a rule, but I can not save my new rule as it is written under the action: `The actions.1.value field is required when actions.1.type is convert_transfer. `. What is the value of this parameter?

![image](https://user-images.githubusercontent.com/6248122/212499964-a48c9130-2797-4d15-9ee7-c6119171a240.png)

Thanks for your help, and especially to JC5 for your wonderful work!

**Comment by JC5** [comment, marked as answer]
Thanks for the compliments!

The parameter you need is described here: https://docs.firefly-iii.org/firefly-iii/pages-and-features/rules/#converting-to-another-transaction-type

**Reply by Oliv4945** [reply]
Oh, I missed it! Thanks :)

**Reply by SkeyeNexus** [reply]
That page is gone. can someone help me?

**Reply by JC5** [reply]
https://docs.firefly-iii.org/references/firefly-iii/rule-actions/#converting-to-another-transaction-type
